.buildkite/al2_cleanup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+source .buildkite/al2env.sh
+
+sudo rm -rf $dir
+./tools/thinpool.sh remove $uniqueID
```

Comment:
Looks like [something is failing](https://buildkite.com/firecracker-microvm/firecracker-containerd-al2/builds/29#0c9f34ba-4a50-4839-9241-045c36b77718/34-36).

Reply:
containerd cleans up the snap devices, but doesn't clean up the thinpool, since our script removes the snap devices and thin pool, we will always expect the snap device cleanup to fail unless containerd failed to clean it up.